src/DependencyInjection/SurfnetStepupU2fExtension.php

```diff
@@ -37,5 +37,9 @@ public function load(array $config, ContainerBuilder $container)
         $container
             ->getDefinition('surfnet_stepup_u2f.yubico.u2f')
             ->replaceArgument(0, $config['app_id']);
+
+        $container
+            ->getDefinition('surfnet_stepup_u2f.value.app_id')
+            ->replaceArgument(0, $config['app_id']);
```

Comment:
Request: please make this service private as it doesn't seem to be needed outside of this bundle.